components/tidb_query_datatype/src/codec/table.rs

```diff
@@ -163,6 +163,13 @@ pub fn encode_row_key(table_id: i64, handle: i64) -> Vec<u8> {
     key
 }
 
+pub fn encode_common_handle(table_id: i64, handle: &[u8]) -> Vec<u8> {
```

Comment:
I guess you can accept `Vec<ScalarValue>` to make this function easier to use. Also prefer to add `_for_test` suffix since it will never be used in non tests.

Reply:
Deleted